ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -70,7 +81,7 @@ def delete_pvc_ui(self, pvc_name):
         self.do_send_keys(self.pvc_loc["search_pvc"], text=pvc_name)
 
         logger.info(f"Go to PVC {pvc_name} Page")
-        self.do_click(self.pvc_loc["pvc_test"])
+        self.do_click(pvc_4_8b[pvc_name])
```

Comment:
is it possible to pass a generic dict and not one that is for a specific version?

Reply:
Generic dict wasn't possible due to locator change from 4.7 and 4.8, but code is modified and is compatible with both 4.7 and 4.8 versions.